packages/apollo-server-errors/src/__tests__/ApolloError.test.ts

```diff
@@ -47,6 +47,28 @@ describe('ApolloError', () => {
         }),
     ).toThrow(/Pass extensions directly/);
   });
+
+  it('provides toJSON method', () => {
+    const error = new ApolloError('My original message', 'A_CODE', {
+      arbitrary: 'user_data',
+    });
+
+    expect(error.toJSON()).toEqual({
+      message: 'My original message',
+      extensions: {
+        code: 'A_CODE',
+        arbitrary: 'user_data',
+      },
+    });
+  });
+
+  it('provides toString method', () => {
+    const error = new ApolloError('My original message', 'A_CODE', {
+      arbitrary: 'user_data',
+    });
+
+    expect(error.toString()).toEqual('My original message');
```

Comment:
That was useful.
I discovered a bug in `graphql-js` for `toString` we should follow `Error.toString` behavior so it should be `ApolloError: My original message`.
```
> e = new Error('test')
> e.name = 'TEST_NAME'
> e.toString()
'TEST_NAME: test'
```
I will fix it in the `16.0.x` release, I don't think it would be a breaking change for both `graphql-js` and AS since I just added `GraphQLError.toString` in `v16.0.0`.

Reply:
Makes sense. If you're planning to change this in graphql-js, maybe just make this test accept either value?  Or we can fix the test in the next graphql-js Renovate PR.